packages/sitecore-jss-vue/src/components/PlaceholderCommon.ts

```diff
@@ -234,7 +234,7 @@ function createRawElement(elem: any) {
     },
   };
 
-  return h(component);
+  return h(component, { data: elem });
```

Comment:
Should we call it "elem" to be more clear about the type of "data"?

Reply:
Good idea 👍